storage/grpc_client.go

```diff
@@ -1229,7 +1229,7 @@ func (r *gRPCReader) Close() error {
 // an attempt to reopen the stream.
 func (r *gRPCReader) recv() (*storagepb.ReadObjectResponse, error) {
 	msg, err := r.stream.Recv()
-	if err != nil && shouldRetry(err) {
+	if err != nil && ShouldRetry(err) {
```

Comment:
Note: ShouldRetry should not be referenced directly here; instead we should be using the configured value from the retry configuration passed through the interface. Will file a separate bug to fix this up. FYI @noahdietz